client/src/pages/Interview.js

```diff
@@ -98,6 +99,10 @@ const Interview = (props) => {
 
   const classes = useStyles();
 
+  const handleFeedbackOpenClose = () => {
+		setFeedbackOpen(!feedbackOpen)
```

Comment:
```suggestion
		setFeedbackOpen(prevState => !prevState)
```

 It's a [best practice](https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous) to depend on previous state rather than current state as React does not guarantee that the state changes are applied immediately.

Reply:
Thank you! That's great to know.